e2e/tests/notebook/notebook-create-r.spec.ts

```diff
@@ -58,22 +56,19 @@ describe('Create R kernel notebook', () => {
     expect(cell3Output).toMatch(/success$/);
   });
 
-  /*Skipping the test below as they will be moved to the new version of e2e test.
-   * Story tracking this effort: https://precisionmedicineinitiative.atlassian.net/browse/RW-8763*/
-
-  test.skip('Duplicate rename delete notebook', async () => {
+  test('Duplicate rename delete notebook', async () => {
     await loadWorkspace(page, workspaceName);
 
     const analysisPage = new WorkspaceAnalysisPage(page);
     await openTab(page, Tabs.Analysis, analysisPage);
 
     // Start clone notebook.
     const cloneNotebookName = `Duplicate of ${rNotebookName}`;
-    await analysisPage.duplicateNotebook(rNotebookName);
```

Comment:
duplicateNotebook() is no longer used, so we should remove it

Reply:
Done